awx/api/serializers.py

```diff
@@ -5070,6 +5070,18 @@ def get_summary_fields(self, obj):
                                     if fval is not None:
                                         job_template_item[field] = fval
                                 summary_fields['job_template'].append(job_template_item)
+                        if fk == 'workflow_job_template_node':
+                            summary_fields['workflow_job_template'] = []
+                            workflow_job_template_item = {}
+                            workflow_job_template_fields = SUMMARIZABLE_FK_FIELDS['workflow_job_template']
+                            workflow_job_template = getattr(thisItem, 'workflow_job_template', None)
+                            if workflow_job_template is not None:
+                                for field in workflow_job_template_fields:
+                                    fval = getattr(workflow_job_template, field, None)
+                                    if fval is not None:
+                                        workflow_job_template_item[field] = fval
+                                summary_fields['workflow_job_template'].append(workflow_job_template_item)
+# to adhere to DRY principles, the above can eventually be combined into a helper method
```

Comment:
The alignment of this comment is strange.

It would be easiest to leave it out.

While it would be nice to combine these, the schedule case has some unique issues, because of that, it works to hold off on it.

Reply:
Comment has been removed